pkg/services/ngalert/ngalert.go

```diff
@@ -289,6 +289,24 @@ func (ng *AlertNG) init() error {
 	ng.AlertsRouter = alertsRouter
 
 	evalFactory := eval.NewEvaluatorFactory(ng.Cfg.UnifiedAlerting, ng.DataSourceCache, ng.ExpressionService, ng.pluginsStore)
+
+	var recordingWriter writer.Writer
```

Comment:
not related to this PR. writer.Writer sounds odd. I think writer interfaces should be declared on scheduler side.